.gitignore

```diff
@@ -2,11 +2,11 @@ secrets.yml
 users.yml
 extra_vars.yml
 *.txt
-instructor*
 .vagrant/*
-ansible.cfg
```

Comment:
Curious to why we are allowing this to be checked in? I have a bunch of personal setting in my ansible.cfg that have no relevancy to others that I wouldn't want to check in. This said I've spent zero time with the new configs in v2.4.